pkg/loadbalancer/proxy.go

```diff
@@ -311,23 +311,34 @@ func proxyUpdateLoadBalancer(ctx context.Context, clusterName string, service *v
 	if err != nil {
 		return fmt.Errorf("error updating configuration Stdout: %s Stderr: %s : %w", stdout.String(), stderr.String(), err)
 	}
-	return waitLoadBalancerReady(ctx, name, 30*time.Second)
+	return waitLoadBalancerReady(ctx, name, 30*time.Second, directConnectivity)
 }
 
-func waitLoadBalancerReady(ctx context.Context, name string, timeout time.Duration) error {
+func waitLoadBalancerReady(ctx context.Context, name string, timeout time.Duration, directConnectivity bool) error {
 	portmaps, err := container.PortMaps(name)
 	if err != nil {
 		return err
 	}
-	port, ok := portmaps[strconv.Itoa(envoyAdminPort)]
-	if !ok {
-		return fmt.Errorf("envoy admin port %d not found, got %v", envoyAdminPort, portmaps)
+
+	var authority string
+	if directConnectivity {
+		ipv4, _, err := container.IPs(name)
+		if err != nil {
+			return err
+		}
+		authority = fmt.Sprintf("%s:%s", ipv4, strconv.Itoa(envoyAdminPort))
```

Comment:
use net.JoinHostPort() to build the authority so it works later for ipv6

Reply:
Done in 5e03531